backend/app/tests/api/routes/documents/test_route_document_info.py

```
@@ -44,13 +46,10 @@ def test_info_reflects_database(
         assert source == target.data
 
     def test_cannot_info_unknown_document(
-        self,
-        db: Session,
-        route: Route,
-        crawler: Route,
+        self, db: Session, route: Route, crawler: Route, api_key_headers: dict[str, str]
```

Ruff (0.12.2)

Redefinition of unused `crawler` from line 10

(F811)